tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,42 @@
+import pdb
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.apps.contacts.regions.settings_form import SettingsForm
+from gaiatest.apps.contacts.regions.contact_form import ContactForm
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+   
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('main')
+        #pdb.set_trace()
+        contacts.tap_settings()
+        self.take_screenshot('main')
+        settings_form = SettingsForm(self.marionette)
```

Comment:
I did the instanciation here because I need to use the functions in SettingsForm, Can I use them with tap_settings ??